local-cli/info/info.js

```diff
@@ -14,17 +14,28 @@ const envinfo = require('envinfo');
 const info = function() {
   const args = Array.prototype.slice.call(arguments)[2];
 
-  try {
-    envinfo.print({
-      packages:
-        typeof args.packages === 'string'
-          ? ['react', 'react-native'].concat(args.packages.split(','))
-          : args.packages,
+  envinfo
+    .run(
+      {
+        System: ['OS', 'CPU', 'Memory', 'Shell'],
```

Comment:
[react-native-bot](https://github.com/hramos/react-native-bot) is set up to look for the old `envinfo` snippet (see [lines 112 - 124](https://github.com/hramos/react-native-bot/blob/master/lib/bot/old_version.rb#L112-L124)). Since we have control over how this data is printed out when it's done within a RN project, we might as well print out some unique string that we can easily grep for.

Currently that is "Packages: (wanted => installed)" but that's fragile. I don't have any suggestion at the moment, other than it should be a string that is unlikely to appear elsewhere in a GitHub issue.

Reply:
oh wow, good catch. 

We can add a custom title above the output in a .then block to look like this:

```
React Native Environment Info:
  System:
    OS: macOS High Sierra 10.13
    CPU: x64 Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
```

Then look for that?

Reply:
Done!

Reply:
OK, I decided this is common enough of an ask to include this in envinfo@5.7.0. This means the --clipboard option will work correctly. 

Updating and adding the title option.